{{ cookiecutter.repo_name }}/docs/source/min_versions.rst

```diff
@@ -0,0 +1,27 @@
+===================================
+Minimum Version of Python and NumPy
+===================================
+
+
+- This project supports at least the minor versions of Python
+  initially released 42 months prior to a planned project release
+  date.
+- The project will always support at least the 2 latest minor
+  versions of Python.
+- support minor versions of ``numpy`` initially released in the 24
+  months prior to a planned project release date or the oldest
+  version that supports the minimum Python version (whichever is
+  higher).
+- The project will always support at least the 3 latest minor
+  versions of NumPy.
+
+The minimum supported version of Python will be set to
+``python_requires`` in ``setup``.  All supported minor versions of
```

Comment:
```suggestion
``python_requires`` in ``setup.py``.  All supported minor versions of
```

Reply:
This was intentionally `setup` as the function _in_ `setup.py`.  The is copy-pasted from the NEP we should keep it as-is here and move discussion of the wording to numpy.